packages/driver/src/cy/commands/actions/click.ts

```diff
@@ -194,17 +198,21 @@ export default (Commands, Cypress, cy: $Cy, state, config) => {
       // once we establish the coordinates and the element
       // passes all of the internal checks
       return $actionability.verify(cy, $el, config, individualOptions, {
+        subjectFn: options.subjectFn || (() => cy.getSubjectFromChain(subjectChain).eq(index)),
```

Comment:
`perform()` is called one element at a time, and each checks actionability individually, so the subjectFn() needs to pull out the element matching the current index after resolving the subjectChain into a set of elements.

This algorithm has some implications. While it fixes the case where the DOM rerenders between clicks, it fails on a case like so:

```
<tr><td><button>Delete Row</button></td></tr>
<tr><td><button>Delete Row</button></td></tr>
<tr><td><button>Delete Row</button></td></tr>

cy.get('button').click({ multiple: true })
```

The DOM changes - not just rerenders, but changes meaningfully - between each invocation of `perform()`. This will result in errors (resolving .get('button') returns fewer elements with each click, so .get('button').eg(2) will end up not receiving an element).

This case was explicitly laid out in the tech brief as "will not fix", but wanted to call it out now that we're here in practice.